check_yaml.py

```
@@ -0,0 +1,80 @@
+    for md_file in sorted(agents_dir.glob("*.md")):
+        issues = check_yaml_frontmatter(md_file)
+        
+        if issues:
+            issue_files.append((md_file.name, issues))
+            print(f"❌ {md_file.name}")
+            for issue in issues:
+                print(f"   - {issue}")
+        else:
+            clean_files.append(md_file.name)
+            
+    print(f"\n\nSummary:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)